docs/2.deploy/20.providers/coolify.md

```diff
@@ -0,0 +1,23 @@
+# Coolify
+
+> Deploy Nitro apps to [Coolify](https://coolify.io).
+
+**Preset:** `coolify`
+
+:read-more{title="coolify.io" to="https://coolify.io"}
+
+::note
+This is the recommended preset for Coolify deployments.
+::
+
+::note
+Integration with this provider is possible with [zero configuration](/deploy#zero-config-providers).
+::
+
+Coolify supports deploying both static and server-side rendered apps with zero configuration.
+
+## Starter templates
+
+If you want to quckly get started with coolify and nitro you can use the public nitro example in the [coolify examples repository](https://github.com/coollabsio/coolify-examples) to get started with from the Coolify dashboard.
```

Comment:
I have explained this in the [Deployment via presets only](https://github.com/nitrojsdev/community/discussions/7) section. Boilerplates are not acceptable because they cause vendor-lockin. We should instead explain all steps here in the docs

Reply:
I replaced this with a `Getting started` section that takes you through the process step by step now.